README.md

```diff
@@ -83,13 +85,13 @@ deno add @radashi-org/radashi
 
 ## FAQ
 
-- **“I need XYZ, but Radashi doesn‘t have it.”**  
```

Comment:
Did CSpell remove these trailing spaces? They are Markdown sensitive

Reply:
No, I have a vscode extension that did it. What do you mean they are markdown sensitive? Want me to revert?

Reply:
In Markdown, two trailing spaces ensure a line break. Without them, this line and the next one will be merged at render-time.

Reply:
Good to know, will revert

Reply:
Still a change for this line because of a non-standard quote char